README.md

```diff
@@ -166,24 +160,37 @@ Expected Output: It will create .terraform directory in XC3/infrastructure/  loc
 7.  Please copy msg_templates in custodian directory on deployed EC2 instance
 
     ```
-    scp -i "keypair.pem" keypair.pem bastion-host-dns:/directory-to-copy-keypair
+    sudo scp -i "keypair.pem" keypair.pem bastion-host-dns:/directory-to-copy-keypair
+    
+    sudo scp -r -i "keypair.pem" ../cloud_custodian_policies/  user@bastion-host-DNS:~/.
+
     ssh -i "keypair.pem" user@bastion-host-DNS
+
     cd directory (where keypair copied in above command)
+
+    sudo scp -r -i "keypair.pem" cloud_custodian_policies/ user@private-ip-ec2:~/.
+
     ssh -i "keypair.pem" user@private-ip-ec2
-    cp -r  ./cloud_custodian_policies/msg_templates/ custodian/lib/python3.x/site-packages/c7n_mailer/msg_templates/
 
+    sudo chown root:root /home/ubuntu/cloud_custodian_policies/msg_templates/*.html.j2
+
+    sudo cp /home/ubuntu/cloud_custodian_policies/msg_templates/*.html.j2 /custodian/lib/python3.8/site-packages/c7n_mailer/msg-templates/
+    
     ```
 
 8.  Please run the following steps on deployed EC2 instance to trigger XC3 lambda functions.
 
     ```
+        source /custodian/bin/activate
+        cd ccloud_custodian_policies
+
      1. custodian run -s s3://${bucket_name}/iam-user --region ${aws_region} iam-user.yml
 
      2. custodian run -s s3://${bucket_name}/iam-role/ --region ${aws_region} iam-role.yml
 
-     3. custodian run -s tagging-compliance --region ${aws_region} eks-tagging.yml --region all
+     3. custodian run -s tagging-compliance --region ${aws_region} eks-tagging.yml
```

Comment:
why we are removing --region all . It is one time command and should be run once

Reply:
When i run the command with this command, it was giving error, so i removed this flag and run the command, it worked fine for one region.